app/(landing)/views/testimonials/custom-tweet.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Tweet } from 'react-tweet/api'
+import {
+  QuotedTweet,
+  TweetBody,
+  TweetContainer,
+  TweetHeader,
+  TweetInReplyTo,
+  TweetInfo,
+  TweetMedia,
+  TweetNotFound,
+  TweetSkeleton,
+  type TwitterComponents,
+  enrichTweet,
+} from 'react-tweet'
+import { unstable_cache } from 'next/cache'
+import { getTweet as _getTweet } from 'react-tweet/api'
+import { Suspense } from 'react'
+
+interface Props {
+  id: string
+}
+
+const getTweet = unstable_cache(async (id: string) => _getTweet(id), ['tweet'], { revalidate: 3600 * 24 })
+
+export default function CustomTweet({ id }: Props) {
+  return (
+    <Suspense fallback={<TweetSkeleton />}>
+      <CustomTweetLoader id={id} />
+    </Suspense>
+  )
+}
+
+async function CustomTweetLoader({ id }: Props) {
+  try {
+    const tweet = await getTweet(id)
+    return tweet ? <TweetComponent tweet={tweet} /> : <TweetNotFound />
```

Comment:
Hmm, maybe better just not to display anything? If some tweets will stop working and we will display "Not found" it will be noticeable, but if we just skip them, no one will notice that we have some issue.

Reply:
1) We should not have any gaps, because it can break the animation
2) By using "NotFound" component we can notice that something is broken

Reply:
1. Are you sure it breaks animation? I think grid will automatically skip not rendered li element. 
2. Honestly we very really have time to check and fix each small issue, like missing tweet.
Can you make it for me? I will be happy to know, when I show it to investors that everything will work, regardless if one user will delete his tweet